### DISCUSSION TITLE
bc-fips 1.0.2.5 not documented on www.bouncycastle.org

### DISCUSSION THREAD

**Discussion by sfc-gh-jzana** [bcgit/bc-java#1678] [Q&A]
Hi there,
  I see a version 1.0.2.5 of bc-fips on [Maven Central](https://central.sonatype.com/artifact/org.bouncycastle/bc-fips/versions) but no mention of it on the bouncycastle website.  I would expect it to be listed under both Downloads and Release notes [here](https://www.bouncycastle.org/download/bouncy-castle-java-fips/#latest).
  Are there release notes available elsewhere?  Should I log an Issue requesting release notes?  In particular, I would like to check if a few security vulnerabilities have been addressed.

Thanks!
Josh

**Comment by jnaithani** [comment]
There is a mention about 1.0.2.5 in https://github.com/bcgit/bc-java/discussions/1574.

Appears to have a fix for an issue reported on IBM AIX.
